compiler/luci/logex/src/FormattedGraph.cpp

```diff
@@ -833,6 +834,16 @@ bool CircleNodeSummaryBuilder::summary(const luci::CircleReshape *node, locop::N
   return true;
 }
 
+bool CircleNodeSummaryBuilder::summary(const luci::CircleResizeNearestNeighbor *node,
+                                       locop::NodeSummary &s) const
+{
+  s.args().append("input", tbl()->lookup(node->input()));
+  s.args().append("size", tbl()->lookup(node->size()));
+  s.args().append("align_corners", node->align_corners() ? "true" : "false");
+  s.state(locop::NodeSummary::State::PartiallyKnown);
```

Comment:
`PartiallyKnown` : do we have anything more to show? this will show `...` at the end, I think.

Reply:
Missed that. Fixed